chapters/introduction.tex

```diff
@@ -72,8 +72,33 @@ \section{Some Definitions}\label{some-definitions}
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Initialization]\index{initialization}
+The simulation starts with solving the initialization problem at the starting time giving values for all variables that are consistent with the result of the flattening.
+\end{definition}
+
+\begin{definition}[Transient analysis]\index{transient analysis}\index{initial value problem!see{transient analysis}}
+Starting from the result of the initialization problem the model is then simulated forward in time.
+This generates values for the variables at different times -- and uses numerical methods for handling the hybrid DAE generated by the flattening.
+\end{definition}
+
+\begin{definition}[Simulation]\index{simulation}
+For a model simulation is the combination of the initialization followed by the transient analysis.
+\end{definition}
+
+\begin{definition}[Translation]\index{translation}
```

Comment:
Would it be better to place this definition right after _flattening_?

Reply:
That would require a forward reference between the definitions (even if simulation is excluded it is still mentioned), and I wanted to avoid that.

Reply:
Sure, it's a tradeoff between avoiding forward references and giving the definitions in the natural chronological order.  Resolving.